aten/src/ATen/native/mps/MPSGraphVenturaOps.h

```diff
@@ -1,23 +1,23 @@
 #pragma once
 #include <MetalPerformanceShadersGraph/MetalPerformanceShadersGraph.h>
 
+#if !defined(__MAC_13_0) && !defined(MAC_OS_X_VERSION_13_0)
+typedef NSUInteger MPSGraphResizeNearestRoundingMode_Ventura NS_TYPED_ENUM;
```

Comment:
Can we use version number instead of "Ventura"? Like MACOS_13?
Also, when you upstream they will ask to replace typedef with "using".

Reply:
I need some generic name for it in all cases. It is being used in the gridSampler API, and some of the enum values are available in 13.0, others in 13.2 and for before 13.0 there were none available 

E.g here I need only one generic enum name in the end: 
```
  nearestRoundingMode:(MPSGraphResizeNearestRoundingMode_Ventura) nearestRoundingMode
```